src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -91,18 +87,18 @@ int64_t HybridPolicyWithFilter(const ResourceRequest &resource_request,
   bool best_is_available = false;
 
   // Step 2: Perform the round robin.
-  auto round_it = round.begin();
-  for (; round_it != round.end(); round_it++) {
-    const auto &node_id = *round_it;
-    const auto &it = nodes.find(node_id);
-    RAY_CHECK(it != nodes.end());
+  size_t round_index = spread_threshold_ == 0 ? spread_scheduling_next_index_ : 0;
```

Comment:
Can we use the saved index no matter what the spread threshold is?

Reply:
Will do it in the separate round robin pr.